ycmd/tests/subcommands_test.py

```diff
@@ -59,19 +59,8 @@ def foo():
 @with_setup( Setup )
 def RunCompleterCommand_GoTo_Clang_ZeroBasedLineAndColumn_test():
   app = TestApp( handlers.app )
-  contents = """
-struct Foo {
-  int x;
-  int y;
-  char c;
-};
-
-int main()
-{
-  Foo foo;
-  return 0;
-}
-"""
+  contents = open( \
```

Comment:
personally, I like the slash to tie the two lines together visually, but I have removed it as advised